improver/utilities/cube_manipulation.py

```diff
@@ -81,18 +81,44 @@ def collapsed(cube: Cube, *args: Any, **kwargs: Any) -> Cube:
     return new_cube
 
 
-def collapse_realizations(cube: Cube) -> Cube:
+def collapse_realizations(cube: Cube, method="mean") -> Cube:
     """Collapses the realization coord of a cube and strips the coord from the cube.
 
     Args:
         cube:
             Input cube
-
+        method:
+            One of "sum", "mean", "median", "std_dev", "min", "max";
+            default is "mean".
     Returns:
         Cube with realization coord collapsed and removed.
     """
-    returned_cube = collapsed(cube, "realization", iris.analysis.MEAN)
+
+    aggregator_dict = {
+        "sum": iris.analysis.SUM,
+        "mean": iris.analysis.MEAN,
+        "median": iris.analysis.MEDIAN,
+        "std_dev": iris.analysis.STD_DEV,
+        "min": iris.analysis.MIN,
+        "max": iris.analysis.MAX,
+    }
+
+    aggregator = aggregator_dict.get(method)
+    if aggregator is None:
+        raise ValueError(f"method must be one of {list(aggregator_dict.keys())}")
```

Comment:
Codecov is flagging that the case of unsupported method is not tested. Could you add a unit test to cover this? I thought there was one before, but I could be mistaken.

Reply:
done

Reply:
Thanks!